src/Neo/Extensions/NeoTokenExtensions.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// NeoTokenExtensions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Neo.Extensions
+{
+    public static class NeoTokenExtensions
+    {
+        private const byte Prefix_Account = 20;
```

Comment:
`Prefix_Account` has defined in `FungibleToken`, Maybe it doesn't need to be defined twice? (change `FungibleToken. Prefix_Account` to an `internal` const value.